responder/jira/jira.go

```diff
@@ -0,0 +1,124 @@
+package jira
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/flanksource/incident-commander/api"
+
+	jira "github.com/andygrunwald/go-jira"
+)
+
+type JiraOptions struct {
```

Comment:
```suggestion
type JiraIssue struct {
```

I think we will need priority in here as well as maybe custom fields

Reply:
👍🏽